README.md

```diff
@@ -80,7 +80,7 @@ To use this mode, your SFCC instance must be configured to allow the origin of t
       "type": "string"
     },
     "ui:extension": {
-      "url": "https://product-selector.amplience.com",
+      "url": "https://product-selector.extensions.content.amplience.net",
```

Comment:
I'm thinking that for the extension to have an origin the extension needs to be registered, so you can add the same origin iframe sandbox param. So it’s a named extension, instead of initiated by a url.

Reply:
You're right. I've amended this and added a bit more clarification.